### DISCUSSION TITLE
Fedora!

### DISCUSSION THREAD

**Discussion by coacharnold1** [navidrome/navidrome#2775] [Q&A]
I'm having issues getting Navidrome running on my Fedora 39 box.  Does anyone have any experience here?  I assume it's a permissions thing,  the service fails out a few secs after starting.

here's what I get 

navidrome.service - Navidrome Music Server and Streamer compatible with Subsonic/Airsonic
     Loaded: loaded (/usr/lib/systemd/system/navidrome.service; disabled; preset: disabled)
    Drop-In: /usr/lib/systemd/system/service.d
             └─10-timeout-abort.conf
     Active: failed (Result: exit-code) since Sun 2024-01-07 18:06:36 EST; 3s ago
   Duration: 96ms
    Process: 363817 ExecStart=/usr/bin/navidrome --configfile /etc/navidrome/navidrome.toml (code=exited, status=1/FAILURE)
   Main PID: 363817 (code=exited, status=1/FAILURE)
        CPU: 97ms

Jan 07 18:06:36 fausto-desktop systemd[1]: navidrome.service: Scheduled restart job, restart counter is at 5.
Jan 07 18:06:36 fausto-desktop systemd[1]: navidrome.service: Start request repeated too quickly.
Jan 07 18:06:36 fausto-desktop systemd[1]: navidrome.service: Failed with result 'exit-code'.
Jan 07 18:06:36 fausto-desktop systemd[1]: Failed to start navidrome.service - Navidrome Music Server and Streamer compatible with Subsonic/Airsonic.

**Comment by tirtawn** [comment]
One suggesion (if you could not resolve it) since I am using ubuntu.

Just use Docker with navidrome. Then you dont have to worry  about missing things since the docker image will take care of it. Search for deluan/navidrome on docker hub.

**Reply by coacharnold1** [reply]
i tried docker and couldn't figure out where to put the config file . It wasn't clear how to set up any of that or how to make it run all the time.

**Reply by tirtawn** [reply]
Watch this youtube first:
https://www.youtube.com/watch?v=7V5UUJlSknY

for running the navidrome on linux.

Notes: Once you have it on docker, you will like it since you dont have to worry about new Ubuntu/Fedora version breaks the apps.

**Comment by coacharnold1** [comment]
thanks all
